docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -7,34 +7,117 @@ Diarrheal Diseases
+   Adapt additional material from GBD capstone and other sources, e.g. [WHO]_,
    [CDC]_, [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
-The relevant ICD codes are...
-
 Modeling Diarrheal Diseases in GBD 2017
 ---------------------------------------
 
+According to the [GBD-2017-YLD-Capstone-Appendix-1]_, "There are no
+major modelling updates from GBD 2016," (p. 93) and "self-reported
+prevalence is the reference category" (p. 88).
+
+Regarding the duration of a bout of diarrhea, "the mean duration was
+the duration in days, an average of 4.3 (4.2 4.4)". (p. 89, based on a paper referenced there).
+For GBD 2017, the remission period was modeled as 5 days.
+
+The GBD 2017 adjusted for seasonal variation in diarrheal disease, but
+we have not attempted to include this variation in Vivarium yet. (p. 89)
+
+There is substantial additional effort in GBD to divide diarrhea
+burden into the aetiologies of diarrhea, but we have not included
+aetiologies in this simple model.
+
 .. todo::
 
    Add relevant detail about diarrheal diseases modeling process from
-   [GBD-2017-YLD-Capstone-Appendix-1]_ and from the CoD Appendix.
+   the CoD Appendix.
 
 Cause Model Diagram
 -------------------
 
-.. todo::
+.. image:: DD_cause_model.svg
+
+
+S: _S_usceptible to diarrheal diseases
+
+I: _I_nfected and currently experiencing a diarrheal disease bout
 
-   Add Vivarium cause model diagram.
 
 Data Descriptions
 -----------------
+	 
+	 
+.. list-table:: State definitions and initialization
+   :widths: 5 30
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - I
+     - Currently has diarrheal disease.
+   * - S
+     - Does not currently have diarrheal disease.
+	 
+.. csv-table:: Prevalence
+   :header: State,Prevalence,Source
+   :widths: 5, 10, 10
+   :stub-columns: 0
+
+   I,prev_302,como
+   S,1-prev_302,NA
+
+prev_302 = get_draws("cause_id", 302, source = "como", measure_id = 5, gbd_round_id = 5)
+
+.. csv-table:: Excess Mortality Rate
+   :header: State,EMR,source
+   :widths: 5, 10, 10
+   :stub-columns: 0
+
+   I,(deaths_302/pop)/prev_302,codcorrect
+   S,0,NA
+
+deaths_302 = get_outputs("cause", cause_id = 302, measure_id = 1, metric_id = 1, gbd_round_id = 5)
+
+.. csv-table:: Disability Weights
+   :header: Severity level, DW (95% CI),Proportion of estimated diarrhea
+   :widths: 10, 20, 10
+   :stub-columns: 0
+
+   Mild,0.0074 (0.049-0.104),.648
+   Moderate,0.188 (0.125-0.264),.289
+   Severe,0.247 (0.164-0.348),.069
+
+Note, the above proportions sum to 1.006. These numbers come from the appendix
+model writeup [GBD-2017-YLD-Capstone-Appendix-1]_. The severity splits come from a
+meta-analysis on severity independent from the DisMod estimates. They splits are then
+applied to the prevalence and incidence estimates, in order to calculate YLDs by sequela and
+etiology.
+
+.. todo::
+
+	Look into what severity splits were actually used.
+
+.. csv-table:: Remission
+   :header: State,Remission,Source
```

Comment:
I think we should have a table of transitions instead of a remission table.  It can include transition from I to S which is this one, remission, and also the transition from S to I, which is incidence.

Reply:
I agree we should change this to a "Transitions" table with incidence as well as remission. I think the rest looks good.